src/Views/Dogs/DogEdit.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import { useParams } from 'react-router-dom';
```

Comment:
You can chain this off of the import above. 
import { useHistory, useParams }  from 'react-router-dom'